src/functions/getTestCentres/framework/handler.ts

```diff
@@ -0,0 +1,33 @@
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { bootstrapLogging, error } from '@dvsa/mes-microservice-common/application/utils/logger';
+import { format, subYears } from 'date-fns';
+import { HttpStatus } from '../../../common/application/api/HttpStatus';
+import createResponse from '../../../common/application/utils/createResponse';
+import Response from '../../../common/application/api/Response';
+import { bootstrapConfig } from './config/config';
+import { findTestCentres } from './application/active-test-centres';
+import { getDate } from './application/get-date';
+
+export async function handler(event: APIGatewayProxyEvent): Promise<Response> {
+  bootstrapLogging('identify active test centres', event);
+
+  const testCentreActiveDate = getDate(event.queryStringParameters, 'testCentreActiveDate');
+  const decommissionDate = getDate(event.queryStringParameters, 'decomissionTimeFrame');
+
+  await bootstrapConfig();
+  try {
+    const activeTestCentres = await findTestCentres(testCentreActiveDate, decommissionDate);
+    const twoYearsTestCentres = await findTestCentres(
+      testCentreActiveDate,
+      format(subYears(new Date(), 2), 'yyyy-MM-dd'),
+    );
```

Comment:
If we use a Promise all here then the requests will be in parallel instead of one after another (might speed up request)
```
const [activeTestCentres, twoYearsTestCentres] = await Promise.all([
   findTestCentres(testCentreActiveDate, decommissionDate),
   findTestCentres(testCentreActiveDate, format(subYears(new Date(), 2), 'yyyy-MM-dd'))
]);
```

Pretty sure this is syntax, should be able to prove by checking the types of the variables